medium-tests/src/test/java/mediumtest/fixtures/ServerFixture.java

```diff
@@ -414,15 +414,25 @@ private void registerSearchIssueApiResponses() {
 
         var allIssues = issuesPerFilePath.values().stream().flatMap(Collection::stream).collect(toList());
 
-        allIssues.forEach(issue -> mockServer.stubFor(get("/api/issues/search.protobuf?issues=".concat(urlEncode(issue.getKey())).concat("&ps=1&p=1"))
+        allIssues.forEach(issue -> {
+          var searchUrl = "/api/issues/search.protobuf?issues=".concat(urlEncode(issue.getKey()))
+            .concat("&componentKeys=").concat(projectKey)
+            .concat("&ps=1&p=1");
+          if (issue.getKey().contains("PR")) {
+            searchUrl = searchUrl.concat("&pullRequest=").concat("pullRequest");
```

Comment:
We force the PR to have the ID `pullRequest`; maybe we could add PR information in the `ServerProjectBranchBuilder`, with a `withPr(String pullRequestId)` method, so we can check if this branch has a PR opened, and in this case, we would concat the parameter?

Reply:
Already commented above, didn't see